docs/set-up-individual-component-css-loading.md

```diff
@@ -166,6 +166,12 @@ Use `yield` to add body content between `<body>` and `</body>`:
   </head>
 ```
 
+### Check stylesheets are loading individually
+
+When the setup is complete, only the component stylesheets required to render the page are requested. For example, the screenshot below shows separate network requests for the `browse`, `lead-paragraph` and `cards` stylesheets:
+
+![Network requests for individual stylesheets](./images/individual-stylesheets.png)
+
```

Comment:
Perhaps it could be helpful to say where the example screenshot is taken from e.g. 

> for example, the screenshot below _displays network activity for a [browse page e.g. benefits](https://www.gov.uk/browse/benefits) and_ shows separate network requests for the `browse`, `lead-paragraph` and `cards` stylesheets.

Reply:
Thanks, makes sense to me, I've made the suggested changes.